src/index.html

```diff
@@ -1,13 +1,468 @@
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+  <body class="page__body">
+    <header class="header" id="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a href="#" class="top-bar-logo">
+              <img
+                src="./icons/logo.svg"
+                alt="Crazylab logo"
+                class="header__logo"
+              >
+            </a>
+
+            <div class="top-bar__icons top-bar-tablet">
+              <div class="top-bar__additional-tablet">
+                <a class="top-bar__specs">SPECS</a>
+                <p class="top-bar__language">
+                  <a class="top-bar__language-en">EN</a> / <a class="top-bar__language-ua">UA</a>
+                </p>
+              </div>
+
+              <a class="top-bar__btn-tablet">Buy</a>
+
+              <a href="#menu" class="icon icon--menu"></a>
+            </div>
+          </div>
+
+          <div class="header__bottom">
+            <h1 class="header__title">Futuristic Wireless Speaker</h1>
+            <p class="header__description">
+              Luna’s performance is balanced and smooth in all frequency ranges which makes the
+              music both naturally pleasant and distinctly more layered.
+            </p>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <aside class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="top-bar menu__top">
+          <a href="#" class="top-bar__logo">
+            <img
+              src="./icons/logo.svg"
+              alt="Crazylab logo"
+              class="menu__logo"
+            >
+          </a>
+
+          <a href="#" class="icon icon--cross"></a>
+        </div>
+
+        <div class="menu__bottom">
+          <nav class="nav menu__nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a href="#our-story" class="nav__link">OUR STORY</a>
+              </li>
+              <li class="nav__item">
+                <a href="#about-us" class="nav__link">ABOUT US</a>
+              </li>
+              <li class="nav__item">
+                <a href="#features" class="nav__link">FEATURES</a>
+              </li>
+              <li class="nav__item">
+                <a href="#questions" class="nav__link">GET IN TOUCH</a>
+              </li>
+            </ul>
+          </nav>
+
+          <div class="menu__additional-mobile">
+            <a class="menu__specs">SPECS</a>
+            <a class="menu__language"> <a class="menu__language-en">EN</a> / <a class="menu__language-ua">UA</a>
+            </a>
+          </div>
+
+          <a class="menu__btn-mobile">Buy</a>
+        </div>
+      </div>
+    </aside>
+
+    <main class="main">
+      <div class="container">
+        <div class="main__content">
+          <nav class="desktop-nav desktop-menu__nav">
```

Comment:
`desktop-menu__nav` is element of parent `desktop-menu`
where is `desktop-menu`?